archivist/assets.py

```diff
@@ -1,6 +1,24 @@
-"""assets interface
+"""Assets interface
+
+   Access to the assets endpoint.
+
+   The user is not expected to use this class directly. It is an attribute of the
+   :class:`Archivist` class.
+
+   For example instantiate an Archivist instance and execute the methods of the class:
+
+   .. code-block:: python
+
+      with open(".auth_token", mode="r") as tokenfile:
+          authtoken = tokenfile.read().strip()
+
+      # Initialize connection to Archivist
+      arch = Archivist(
+          "https://soak-0-avid.engineering-k8s-stage-2.dev.wild.jitsuin.io",
```

Comment:
Please change all references to soak into references to the rkvst.poc.jitsuin.io host

Reply:
fixed